File: dotnet/Models/Seller.cs
namespace EmbeddedPayments.Models;

public class Seller
{
    public string Id { get; set; } = string.Empty;
    public string Name { get; set; } = string.Empty;
    public double PlatformFeeRate { get; set; }
    public string Description { get; set; } = string.Empty;
    public string ProPayAccountNumber { get; set; } = string.Empty;
}

public class SellersData
{
    public List<Seller> Sellers { get; set; } = new();
}

File: java/src/main/java/com/globalpayments/example/models/Seller.java
package com.globalpayments.example.models;

public class Seller {
    private String id;
    private String name;
    private double platformFeeRate;
    private String description;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public double getPlatformFeeRate() {
        return platformFeeRate;
    }

    public void setPlatformFeeRate(double platformFeeRate) {
        this.platformFeeRate = platformFeeRate;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}
